vault/audit_broker_test.go

```diff
@@ -256,3 +260,58 @@ func TestAuditBroker_Register_MultipleFails(t *testing.T) {
 	require.Error(t, err)
 	require.EqualError(t, err, "vault.(AuditBroker).Register: backend already registered 'b2-no-filter'")
 }
+
+func BenchmarkAuditBroker_File_Request_DevNull(b *testing.B) {
```

Comment:
Nit: A godoc would be great for this :)

Reply:
doh! I moved it from an older test in `file` backed, I shall add one 😄